web3/_utils/method_formatters.py

```diff
@@ -472,7 +473,7 @@ def get_request_formatters(
 
 def get_result_formatters(
     method_name: Union[RPCEndpoint, Callable[..., RPCEndpoint]]
-) -> Dict[str, Callable[..., Any]]:
+) -> Dict[str, Callable[..., TReturn]]:
```

Comment:
I'm not sure these actually improve the situation any.  The idea behind `TypeVar` is that it lets you setup linkages between the various types for something.  Without any of the calling args using `TReturn` there is no way to anchor what the type should be so I suspect this is roughly equivalent to `Any`